recipes/tvb-framework/meta.yaml

```diff
@@ -0,0 +1,67 @@
+
+package:
+  name: tvb-framework
+  version: "1.5.4"
+
+source:
+  git_rev: 1aaf90c67450dfe535a7f76002f203218de330b4
+  git_url: https://github.com/the-virtual-brain/tvb-framework.git
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install
+  skip: True  # [py>27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - allensdk
+    - apscheduler
+    - BeautifulSoup4
+    - cfflib
+    - cherrypy
+    - coverage
+    - formencode
+    - genshi
+    - h5py
+    - lxml
+    - minixsv
+    - mod_pywebsocket
+    - Pillow
+    - psutil
+    - simplejson
+    - sqlalchemy  0.7.8
+    - sqlalchemy-migrate  0.7.2
+    - tvb-data
+    - tvb-library
+  run:
+    - python
+
+test:
+  imports:
+    - tvb
+    - tvb.interfaces
```

Comment:
These tests don't actually do anything useful, as illustrated by your tests passing when all these dependencies are missing. It'd be better to import some path that actually runs some code, e.g. `tvb.interfaces.run` or something.